docs/content/references/sui-api/beta-graph-ql.mdx

```diff
@@ -20,6 +20,15 @@ Currently, there is one available playground for Sui GraphQL Service:
 
 This instance runs the GraphQL IDE, an interactive IDE with autocomplete support and in-built documentation.
 
+The endpoint can also be queried using `curl` by passing the query to the call:
+```shell
+curl -X POST https://graphql-beta.mainnet.sui.io \
+     --header "Content-Type: application/json" \
+     --data '{ 
+       "query": "query { chainIdentifier }"
+     }'
```

Comment:
Thanks for getting on this.
I believe we should show using variables and fragments too for completeness.

Reply:
Sure, but shouldn't that go into the examples folder?

Reply:
I think it makes sense to pay special attention to variables here to show to provide the actual parameters for variables in a query.  I'm not sure if there's an equivalent consideration for fragments (I'm not aware of one, but I'm not the expert here).

Reply:
@oxade @amnn added variables (with @wlmyng's help). No idea about fragments, if @oxade you know how to, please share :D